samples/Digipolis.Web.SampleApi/Startup.cs

```diff
@@ -32,6 +39,7 @@ public void ConfigureServices(IServiceCollection services)
         {
             // Add framework services.
             services.AddMvc()
+                .AddNewtonsoftJson(options => options.SerializerSettings.Initialize()) // set default Digipolis json serializer settings
```

Comment:
Thisi s a workaround for not being able to use MvcOptions anymore.
Making the initialize public, also lets the consumer hook on the JsonSerializerSettings with an action.